strabo/static/js/map.js

```diff
@@ -37,7 +37,24 @@ $(document).ready(function(){
     //set_map_click(map);
 
     set_geolocation(map);
+
+});
+
+$(document).keyup(function(e) {
+    if (e.keyCode==27) { //if ESC key is hit
```

Comment:
I am intrigued by that suggestion but I think that the comment in sufficient in this case - because we are using a common keycode. I think that the variable assignment complicates things